.gitattributes

```diff
@@ -0,0 +1,8 @@
+/.gitattributes export-ignore
+/.github export-ignore
+/.gitignore export-ignore
+/.php_cs.dist export-ignore
+/phpstan.neon.dist export-ignore
+/phpstan.neon.lowest.dist export-ignore
+/phpunit.xml.dist export-ignore
+/tests export-ignore
```

Comment:
Missing following paths:
- doc
- features

Reply:
disagree on doc, it might be useful to have them locally :)
for the features one, I thinkI thought they were needed so the PHPUnit work properly but seems not. so yes you can put it back @toofff, sorry

Reply:
@vincentchalamon could you look ate the CI error?
seems like a decryption error with docusign secrets

Reply:
Right for keeping `doc` directory as it's required by the README.md file.
`features` directory can be removed from release process cause we don't need it (neither than phpunit) on a release.

Reply:
CI is failing because the passphrase (`DOCUSIGN_RSA_PASSPHRASE`), required to decode the DocuSign certificate, is not shared with forks. It's on your side @GregoireHebert, I don't have access to it...

Reply:
I will watch @vincentchalamon ;)